pxtlib/gallery.ts

```diff
@@ -63,6 +64,15 @@ namespace pxt.gallery {
 
         return {};
     }
+    export function parseSimThemeJSON(md: string): Partial<pxt.PackageConfig> {
+        const pm = /```simtheme\s+([\s\S]*?)```/i.exec(md);
```

Comment:
Just curious, what does "pm" stand for?

Reply:
is it parse markdown??

Reply:
That's what I was thinking, but wasn't sure.

Reply:
it's just from copy / paste of the few functions above it, no thinking involved :) if I had to guess the `package` function was probably added first, and `m` is commonly used as a variable name for `Regex.exec` results as a shorthand for `match` (similar to `i` in loops, etc) so it probably was originally `package match`, but just not changed to match previous, similar code.